app/gateway/2.8.x/plan-and-deploy/systemd.md

```diff
@@ -136,7 +136,7 @@ To use the [injection system](/gateway/{{page.kong_version}}/reference/configura
 Environment="KONG_NGINX_HTTP_OUTPUT_BUFFERS=4 64k"
 ```
 
-### Customize Kong's Nginx instance using `--nginx-conf`
+### Customize Kong's Nginx instance using --nginx-conf
```

Comment:
This gets turned into one dash in the output:
![Screen Shot 2022-03-10 at 11 07 37 AM](https://user-images.githubusercontent.com/54370747/157736817-15f6cf80-db2d-47f3-a2ff-977322c83831.png)
 
Maybe try this:
```suggestion
### Customize Kong's Nginx instance using &ndash; &ndash;nginx-conf
```